src/Breadcrumb/BreadcrumbItem.tsx

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import classnames from "classnames";
+
+export type BreadcrumbItemProps = React.PropsWithChildren<
+    React.DetailedHTMLProps<React.LiHTMLAttributes<HTMLLIElement>, HTMLLIElement> & {
+        /**
+         * The link to where it navigates to. This is used to enable openning the link in new tab.
+         * Additionally, you can access it in the event passed with the onNavigate callback
+         */
+        href?: string;
+        /** Event handler triggered when the links is clicked */
+        onNavigate?: React.MouseEventHandler<HTMLAnchorElement>;
+    }
+>;
+
+export const BreadcrumbItem: React.FC<BreadcrumbItemProps> = React.memo(({ href, onNavigate, ...props }: BreadcrumbItemProps) => {
+    const [active, setActive] = React.useState<boolean>(false);
+    const [className, setClassName] = React.useState<string>("breadcrumb-item");
+
+    React.useEffect(() => setActive(props["data-active"] !== undefined && JSON.parse(props["data-active"])), [props["data-active"]]);
```

Comment:
JSON.parse("") will throw an error, to avoid i suggest to replace
 ```typescript props["data-active"] !== undefined```
```suggestion
    React.useEffect(() => setActive(props["data-active"] && JSON.parse(props["data-active"])), [props["data-active"]]);
```

Reply:
Fix in a different way